README.md

```diff
@@ -21,16 +21,17 @@ For more information please refer to the [documentation](https://krill.docs.nlne
 
 # Changelog
 
-## 0.10.0 RC2 'Hush'
+## 0.10.0 RC3 'Hush'
```

Comment:
Perhaps it's better to do this as a part of a separate release-0.10.0-rc3 issue. It's not critical of course, but the krill version is still 0.10.0-rc2 - so technically this may be premature

Reply:
The idea was to set the Cargo.toml version to rc3 in the rc3 branch that this PR merges into.

Reply:
works for me!